contracts/src/v0.8/tests/VRFLoadTestExternalSubOwner.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/LinkTokenInterface.sol";
+import "../interfaces/VRFCoordinatorV2Interface.sol";
+import "../VRFConsumerBaseV2.sol";
+
+/**
+ * @title The VRFLoadTestExternalSubOwner contract.
+ * @notice Allows making many VRF V2 randomness requests in a single transaction for load testing.
+ */
+contract VRFLoadTestExternalSubOwner is VRFConsumerBaseV2 {
+  VRFCoordinatorV2Interface public immutable COORDINATOR;
+  LinkTokenInterface public immutable LINK;
+
+  uint256 public s_responseCount;
+  address s_owner;
+
+  constructor(address _vrfCoordinator, address _link) VRFConsumerBaseV2(_vrfCoordinator) {
+    COORDINATOR = VRFCoordinatorV2Interface(_vrfCoordinator);
+    LINK = LinkTokenInterface(_link);
+    s_owner = msg.sender;
+  }
+
+  function fulfillRandomWords(uint256, uint256[] memory) internal override {
+    s_responseCount++;
+  }
+
+  function requestRandomWords(
+    uint64 _subId,
+    uint32 _callbackGasLimit,
+    uint16 _requestConfirmations,
+    uint32 _numWords,
```

Comment:
Are these arguments necessary to have in this method? Are we thinking we'd vary `_callbackGasLimit`, `_requestConfirmations`, etc. for a load test, or is it less boilerplate-y to have them as constants like `COORDINATOR` and `LINK` if they're not gonna change much. We'd probably want to load test on diff keyhashes though, so makes sense to have the keyhash as an argument.